students/slawinski_amadeusz/lesson_05_lists/age_calculator.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+print("How many people ages you have?")
+peoples = int(input())
+
+ages = [int(input()) for x in range(peoples)]
+
+adults = [x for x in ages if x >= 18]
+kids = [x for x in ages if x < 18]
+
+average = 0
+for i in adults:
+    average = average + i
+average = average / len(adults)
```

Comment:
a co będzie jak użytkownik nie poda dorosłych?
Przydatny może być warunek `if adults:`, który równoważny jest do: `if adults is not None and len(adults) > 0:`